### ISSUE TITLE
Can't delete user in the admin panel: database table is locked

### ISSUE THREAD

**Issue by sbstp** [go-gitea/gitea#1735]
- Gitea version (or commit ref): 1.1.1 built with: bindata, sqlite
- Git version: 2.7.4
- Operating system: Ubuntu 16.04, amd64
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant (not admin)
- Log:
```
2017/05/16 11:23:31 [I] Run Mode: Production
2017/05/16 11:23:31 [I] Listen: http://localhost:3000
2017/05/16 11:24:07 [...uters/admin/users.go:257 DeleteUser()] [E] DeleteUser: ExternalLoginUser: database table is locked
2017/05/16 11:24:14 [...uters/admin/users.go:257 DeleteUser()] [E] DeleteUser: ExternalLoginUser: database table is locked
2017/05/16 11:24:32 [...uters/admin/users.go:257 DeleteUser()] [E] DeleteUser: ExternalLoginUser: database table is locked
2017/05/16 11:24:46 [...uters/admin/users.go:257 DeleteUser()] [E] DeleteUser: ExternalLoginUser: database table is locked
2017/05/16 11:26:26 [...uters/admin/users.go:257 DeleteUser()] [E] DeleteUser: ExternalLoginUser: database table is locked
```
## Description
I get a 500 error when I try to delete a user from the control panel. I've seen other similar issues with table lock problems, but I might as well document this particular instance.

**Comment by gravemalte** [comment]
This is already fixed by #1321. You can build Gitea on your own or you wait until 1.2.0 is released. :)

**Comment by sbstp** [comment]
Ah thanks. My bad.
